### DISCUSSION TITLE
[ERR] failed to load provider keys got yaml: unmarshal errors:   line 1: cannot unmarshal !!str `/home/o...` into sources.Provider

### DISCUSSION THREAD

**Discussion by knowthetech** [projectdiscovery/nuclei#3802] [General]
[ERR] failed to load provider keys got yaml: unmarshal errors:
  line 1: cannot unmarshal !!str `/home/o...` into sources.Provider

**Comment by ehsandeep** [comment]
@knowthetech update your nuclei to latest version - https://github.com/projectdiscovery/nuclei/issues/3771

**Reply by knowthetech** [reply]
Updated and after that i am getting  this error
![Uploading IMG-20230609-WA0010.jpg…]()

**Comment by knowthetech** [comment]
![IMG-20230609-WA0010](https://github.com/projectdiscovery/nuclei/assets/52136037/91b5104b-88db-4658-b335-701ef227b244)

**Reply by MetzinAround** [reply]
https://askubuntu.com/a/421795. The CLIBC error might be related to this ask ubuntu error. See if that fixes it and let us know if it doesn't.

**Reply by knowthetech** [reply]
Not got fix by that so i configured already build version and it worked for me
